paddlescience/network/network_fc.py

```diff
@@ -57,12 +57,15 @@ def __init__(self,
         self._weights_attr = [None for i in range(num_layers)]
         self._bias_attr = [None for i in range(num_layers)]
 
-        if activation == 'sigmoid':
-            self.activation = F.sigmoid
-        elif activation == 'tanh':
-            self.activation = paddle.tanh
+        act_str = {'sigmoid': F.sigmoid, 'tanh': paddle.tanh}
+        if isinstance(activation, str) & (activation in act_str):
```

Comment:
> and is a Logical AND that returns True if both the operands are true whereas ‘&’ is a bitwise operator in Python that acts on bits and performs bit by bit operation.
Note: When an integer value is 0, it is considered as False otherwise True when using logically.

python中逻辑运算符一般用and, 此处刚好为bool值，所以&也正确，不过建议遵循惯用写法

Reply:
done